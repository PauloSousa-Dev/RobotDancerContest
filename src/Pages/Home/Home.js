export function Home() {
  return <main>Hello</main>;
}
