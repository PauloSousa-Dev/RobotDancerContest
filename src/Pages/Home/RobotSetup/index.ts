export { RobotSetup, NUMBEROFROBOTS } from "./RobotSetup";
