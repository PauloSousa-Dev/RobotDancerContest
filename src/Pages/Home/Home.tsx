import { TeamsProvider } from "@/Contexts";
import { RobotSetup } from "./RobotSetup";

const Home = () => {
  return (
    <TeamsProvider>
      <h1>Robo-Dance Competition</h1>
      <RobotSetup />
    </TeamsProvider>
  );
};

export { Home };
