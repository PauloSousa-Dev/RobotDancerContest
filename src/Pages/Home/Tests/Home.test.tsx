import React from "react";
import { render, screen } from "@testing-library/react";
import { describe, expect, test } from "@jest/globals";
import { Home } from "../Home";

describe("Home component", () => {
  test("should render correctly", () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
