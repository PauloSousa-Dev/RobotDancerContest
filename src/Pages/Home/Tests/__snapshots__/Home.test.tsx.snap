// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should render correctly 1`] = `
<div>
  <h1>
    Robo-Dance Competition
  </h1>
  <div
    class="wrapper"
  >
    <h2>
      Set Up Teams
    </h2>
    <div
      class="inputWrapper"
    >
      <input
        placeholder="Team 1 Name"
        type="text"
        value=""
      />
      <input
        placeholder="Team 2 Name"
        type="text"
        value=""
      />
    </div>
  </div>
</div>
`;
